include/libstatistics_collector/topic_statistics_collector/topic_statistics_collector.hpp

```diff
@@ -78,22 +82,26 @@ class TopicStatisticsCollector<rmw_message_info_t,
 public:
   TopicStatisticsCollector() = default;
 
-  virtual ~TopicStatisticsCollector() = default;
+  ~TopicStatisticsCollector() override = default;
```

Comment:
I would agree with keeping this virtual, especially since it's the base class and we have nothing to override

Reply:
Well while syntactically it will be less confusing to keep it virtual, semantically it will affect nothing since there are only a default constructors in the base classes.
I will change it back to virtual.